src/data_access/FileFolderDataAccessObject.java

```diff
@@ -131,4 +132,17 @@ public Folder get(String folderName) {
     public HashMap<Integer, Recipe> getrecipeMap(String foldername) {
```

Comment:
this function I did some changes in my branch

Reply:
hi, but you are looking at the outdated version. if you check out the newer commits, this should be fixed.

Reply:
yes  we can use yours now